src/sentry/api/endpoints/project_performance_issue_settings.py

```diff
@@ -16,6 +16,7 @@
     PerformanceConsecutiveDBQueriesGroupType,
     PerformanceConsecutiveHTTPQueriesGroupType,
     PerformanceDBMainThreadGroupType,
+    PerformanceDurationRegressionGroupType,
```

Comment:
Just realized but i think this should be set for `PerformanceP95EndpointRegressionGroupType` instead because that will be the release issue type.

Reply:
ah okay, I can switch to the final issue type 👍